rope/base/prefs.py

```diff
@@ -206,6 +214,8 @@ class Prefs:
             Can only be set in config.py.
         """),
     )
+    autoimport: AutoimportPrefs = field(
+        default_factory=lambda: AutoimportPrefs(), description="Preferences for Autoimport")
```

Comment:
Is the `lambda` necessary here? Can't it just be: `default_factory=AutoimportPrefs`?

Reply:
fixed in a05f7d53